src/java.desktop/share/classes/java/awt/Robot.java

```diff
@@ -395,12 +454,28 @@ public synchronized Color getPixelColor(int x, int y) {
     }
 
     /**
-     * Creates an image containing pixels read from the screen.  This image does
-     * not include the mouse cursor.
+     * Creates an image containing pixels read from the screen.
+     * <p>
+     * If the desktop environment requires that permissions be granted
+     * to capture screen content, and the required permissions are not granted,
+     * then a {@code SecurityException} may be thrown,
+     * or the contents of the returned {@code BufferedImage} are undefined.
+     * <p>
+     * The {@link #revokeScreenCapturePermission()} can be used to revoke
+     * a previously granted permission.
+     * <p>
```

Comment:
I would like to completely separate any decision on adding revokeScreenCapturePermission from
this spec. relaxation.
So we should remove the reference to it here into a separate PR about adding that API although I'm still not convinced we should add it.

Reply:
Removed from this PR